FrameworkSystem/private/standardLogging/Formatter/JsonFormatter.py

```diff
@@ -0,0 +1,22 @@
+"""
+Formatter as JSON structure
+"""
+
+
+from DIRAC.FrameworkSystem.private.standardLogging.Formatter.BaseFormatter import BaseFormatter
+from jsonlogger import JsonFormatter as libJsonFormatter
```

Comment:
just for style: invert the order of the two lines above. 

And more on this file: add `__RCSID__` and add one empty line at the end.

Reply:
I thought we agreed on giving up the RCSID since we do not use SVN anymore ?